InAppPurchasesAtlasAppServices/FoodieFolio/FoodieFolio/Managers/StoreKitManager.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  StoreKitManager.swift
+//  FoodieFolio
+//
+//  Created by Mar Cabrera on 31/08/2023.
+//
+
+import Foundation
+import StoreKit
+
+/// `StoreKitManager` is responsible for managing in-app purchases and product information.
+///  NOTE:
+class StoreKitManager: ObservableObject {
```

Comment:
I'm missing some more detailed explanation of the steps to retrieve products from the store kit and complete a purchase